packages/dashboard/src/components/widgets/primitives/input/hooks/useWritableSiteWiseProperty.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  DescribeAssetCommand,
+  IoTSiteWiseClient,
+  ListTagsForResourceCommand,
+  ErrorDetails,
+} from '@aws-sdk/client-iotsitewise';
+import { useContext } from 'react';
+import { ClientContext } from '~/components/dashboard/clientContext';
+
+export const WRITABLE_TAG = 'iot-app-kit:writable';
```

Comment:
Blocking: This is a 1-way door. We should not publicly release this without a principle engineer review and likely a security review.

Reply:
Good callout. Why do you think this is a 1-way door? I'll reach out to app sec to kick off a review for this.